.gitignore

```diff
@@ -1,19 +1,22 @@
-# Compiled python code
-*.pyc
-
-# osx files
-.DS_Store
-
-# sass cache
-.sass-cache
-.elasticbeanstalk
-
-# Distribution
-dist
-django_raster.egg-info
-
-# Docs build
-docs/_build
-
-# Coverage output
-.coverage
+# Compiled python code
+*.pyc
+
+# osx files
+.DS_Store
+
+# sass cache
+.sass-cache
+.elasticbeanstalk
+
+# Distribution
+dist
+django_raster.egg-info
+
+# Docs build
+docs/_build
+
+# Coverage output
+.coverage
+
+.idea/
+.idea
```

Comment:
Correct me if I am wrong, but I don't think the change on gitignore is necessary. Can you remove the changes?

Reply:
I can - but they don't do any harm for future developers using PyCharm?